docs/Configuration.md

```diff
@@ -1,206 +1,206 @@
-# Configuration File
+# Configuration
 
-## Overview
+Cayley can be configured using configuration file written in YAML / JSON or by passing flags to the `http` command line.
```

Comment:
It's not specific to `http`, it works for other commands as well: `repl`, `load`, `init`, `dump`.